proposals/lip-0070.md

```diff
@@ -197,8 +196,7 @@ claimRewardsParams = {
 
 ```python
```

Comment:
Similar to my comment for the unlock command in LIP0057, I would completely remove pseudocode and say in words that we don't perform any additional verification.